### ISSUE TITLE
gts@3 issues related to generated code

### ISSUE THREAD

**Issue by JustinBeckwith** [googleapis/gapic-generator-typescript#702]
I have a release candidate of `gts@3` sitting here:
https://www.npmjs.com/package/gts/v/3.0.0-alpha.1

Still working on a changelog, but it's mostly upgrading all the dependencies.  I have a PR out on all the repos, and I'm seeing runs like this:
https://github.com/googleapis/nodejs-vision/pull/823/checks?check_run_id=1078358969

The pattern looks to match:
```
/home/runner/work/nodejs-vision/nodejs-vision/src/v1/image_annotator_client.ts
##[warning]  22:3   warning  'CallOptions' is defined but never used         @typescript-eslint/no-unused-vars
##[error]  32:17  error    Require statement not part of import statement  @typescript-eslint/no-var-requires
```

Ideally we'd address both of these before rolling out the 3.x release.  Right now I'm just poking around for what's going to cause trouble :)

**Comment by alexander-fenster** [comment]
Got it, will fix!

**Comment by alexander-fenster** [comment]
Fixed in #745
